bugsnag-plugin-android-ndk/src/androidTest/java/com/bugsnag/android/ndk/migrations/EventMigrationV10Tests.kt

```diff
@@ -136,7 +136,7 @@ class EventMigrationV10Tests : EventMigrationTest() {
                     "stacktrace" to listOf(
                         mapOf(
                             "frameAddress" to "0xfffffffe",
-                            "lineNumber" to 4194967233L,
+                            "lineNumber" to "0xfa0a1ec1",
```

Comment:
Should these tests be changed? The existing legacy data on disk will be integers, not hex encoded strings...

Reply:
The binary files are only converted to JSON as part of delivery, so the legacy events in these tests would go through the new JSON encoding (resulting in legacy data having hex encoded strings, and not integers).